drivers/dai/intel/ssp/Kconfig.ssp

```diff
@@ -9,6 +9,7 @@ config DAI_INTEL_SSP
 	bool "Intel I2S (SSP) Bus Driver for Dai interface"
 	default $(dt_compat_enabled,$(DT_COMPAT_DAI_INTEL_SSP))
 	select DMA
+	depends on PM_DEVICE_RUNTIME
```

Comment:
```	depends on PM_DEVICE_RUNTIME
```
I don't think make it a hard requirement ... the power managment subsytem is optional.
The calls to put/get will not do anything when device runtime PM is disabled.

Reply:
It is. Turn on/off is invoked by a host IPC message containing drvdata, which is set as a part of the flow. If device is turned on once at the beginning, it won't have proper drvdata set.